openstack/cronus/templates/cronus/secrets.yaml

```diff
@@ -2,16 +2,22 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: cronus
+  name: cronus-secret
   namespace: {{ .Release.Namespace }}
-  annotations:
-    cloud.sap/skip-template: "true"
   labels:
     app: cronus
     chart: cronus
     release: cronus
-    heritage: {{ .Release.Service }}
 type: Opaque
 data:
-  config.yaml: {{ include "cronus/templates/cronus/_config.yaml.tpl" . | b64enc }}
+  SENTRY_DSN: {{ .Values.cronus.sentryDsn | b64enc }}
+  RABBITMQ_URI: {{ printf "amqp://%s:%s@%s/" .Values.rabbitmq.users.default.user .Values.rabbitmq.users.default.password .Values.rabbitmq.host | b64enc }}
+  {{- $host := printf "%s.%s.%s:5672" "hermes-rabbitmq-notifications" $.Values.global.region $.Values.global.tld }}
+  RABBITMQ_URL: {{ printf "amqp://%s:%s@%s/" .Values.rabbitmq_notifications.users.default.user .Values.rabbitmq_notifications.users.default.password $host | b64enc }}
```

Comment:
I think this wont work on qa-de-2 as there used to be the logic:
```
{{ if .Values.config.cronusAuditSink.host }}{{ .Values.config.cronusAuditSink.host }}{{ else }}{{ $host }}{{ end }}
```

I think only qa-de-2 has `cronusAuditSink`. But looking at the value, and maybe it does work on qa-de-2 as the `cronusAuditSink` is similar. Lets try it like this, but just keep in mind that for qa-de-2 it could fail. In which case we can think of using a `hermesHost` in the values file or something.